runtime/flang/CMakeLists.txt

```diff
@@ -630,20 +579,24 @@ target_include_directories(flang_shared
   )
 
 # Make sure the compiler is built before we bootstrap
-add_dependencies(flang_static 
+add_dependencies(flang_static
   flang1
   flang2
   )
 
 # Make sure the compiler is built before we bootstrap
-add_dependencies(flang_shared 
+add_dependencies(flang_shared
   flang1
   flang2
   )
 
-target_compile_options(flang_static PRIVATE -fPIC)
-
-target_compile_options(flang_shared PRIVATE -fPIC)
+if (NOT MSVC)
```

Comment:
Same as above

Reply:
Same as in the other commit of this refactored PR.